safe/definitions.py

```diff
@@ -559,18 +576,24 @@
     ]
 }
 
-exposure_population = {
-    'key': 'population',
-    'name': tr('Population'),
+exposure_people_in_building = {
+    'key': 'people_in_building',
+    'name': tr('People in buildings'),
     'description': tr(
-        'The <b>population</b> describes the people that might be '
-        'exposed to a particular hazard.'),
-    'notes': exposure_people_in_building['notes'],
-    'actions': exposure_people_in_building['actions'],
+        'The <b>people in buildings</b> exposure data is an experimental '
+        'data set that assigns the population of a specific administrative '
```

Comment:
I don't think it makes sense to refer to the 'dataset' (user have their own PIB data which may not be experimental).

The <b>people in buildings</b> exposure data assigns the population of a specific administrative '